index.html

```diff
@@ -4,6 +4,7 @@
     <title>מפתח התקציב - חיפוש</title>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <base href="/">
```

Comment:
Will this work in production as well (as the site is hosted on https://next.obudget.org/app/search/)?
Perhaps we need this to be configurable?